runner/check-aws-ecs-tasks.js

```diff
@@ -17,23 +17,30 @@ async function main() {
 
   const command = new ListTasksCommand(params)
 
+  const data = await client.send(command)
+
   try {
-    const data = await client.send(command)
     if (data.$metadata.httpStatusCode > 399) {
       throw Error(data.$metadata.httpStatusCode)
     } else {
       if (data.taskArns.length > 0) {
+        console.log('Running tasks')
         console.log(data.taskArns)
       } else {
-        console.log(0)
+        console.log('No running tasks found')
       }
     }
   } catch (error) {
```

Comment:
Can any of this code within the try-catch actually throw an exception, other than the exception that is manually thrown on line 24?  Can we just log there instead and remove the try-catch?

Reply:
Also, this whole function is wrapped in a catch from how main() is run, so even if we kept throwing there the try-catch here still seems unnecessary